subprojects/gradle/instrumentation-tests/src/integTest/kotlin/com/avito/instrumentation/executing/TestExecutorIntegrationTest.kt

```diff
@@ -0,0 +1,95 @@
+package com.avito.instrumentation.executing
+
+import com.avito.instrumentation.configuration.target.TargetConfiguration
+import com.avito.instrumentation.configuration.target.scheduling.quota.QuotaConfiguration
+import com.avito.instrumentation.createStubInstance
+import com.avito.instrumentation.report.listener.FakeTestReporter
+import com.avito.instrumentation.reservation.client.kubernetes.KubernetesReservationClient
+import com.avito.instrumentation.reservation.client.kubernetes.createStubInstance
+import com.avito.instrumentation.reservation.devices.provider.KubernetesDevicesProvider
+import com.avito.instrumentation.reservation.request.Device
+import com.avito.instrumentation.reservation.request.Reservation
+import com.avito.instrumentation.reservation.request.createStubInstance
+import com.avito.instrumentation.suite.model.TestWithTarget
+import com.avito.report.model.TestStaticDataPackage
+import com.avito.report.model.createStubInstance
+import com.avito.runner.service.worker.device.adb.Adb
+import com.avito.runner.service.worker.device.adb.AdbDevicesManager
+import com.avito.utils.logging.CILogger
+import com.avito.utils.logging.FakeCILogger
+import com.avito.utils.logging.commonLogger
+import org.junit.jupiter.api.Test
+import org.junit.jupiter.api.io.TempDir
+import java.io.File
+
+internal class TestExecutorIntegrationTest {
+
+    private val logger = FakeCILogger()
+
+    @Test
+    fun `executor - finished - no requests`(@TempDir tempDir: File) {
+        val executor = createTestExecutor(logger = logger)
+
+        executor.execute(
+            application = null,
+            testApplication = File(""),
+            testsToRun = listOf(),
+            executionParameters = ExecutionParameters.createStubInstance(),
+            output = tempDir
+        )
+
+        // no assertion needed: just stuck in runBlocking forever before fix
+    }
+
+    @Test
+    fun `executor - finished - problem with deployment`(@TempDir tempDir: File) {
```

Comment:
Could you make the problem explicit I don't understand that problem in the test body

Reply:
yep, i changed test names to be more clear